kb/section_subjdomain_of_natural_language_texts/examples/dialogue_dicts/dialogue_dict_4.scs

```diff
@@ -0,0 +1,488 @@
+l_en_we
+    <= nrel_paradigm: ...
+    (*
+        -> rrel_gramm_number_plural: rrel_gramm_case_subjective: rrel_gramm_person_first: ...
+        (*
+            -> rrel_standard: [WE];;
+        *);;
+    *);
+    <- lexeme_taigen; lexeme_minimized;
+    <- lexeme_compressed; lexeme;
+    <- lexeme_informational;lexeme_variable;
+    <- lexeme_arbitrary; <- lexeme_qualitative;
```

Comment:
codestyle

Reply:
What is the code style? Without second '<-' per a line or one node per a line? Please give an example